### PR TITLE
[RELEASE] Re-brands the module to from PayEx_Core to SwedbankPay_Core

### PR DESCRIPTION
**Pull request by shahriar-vdc**

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Block/Adminhtml/System/Config/Fieldset/Payment.php  7
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-core/pullRequest?prid=5236970&bid=17165678)

File: Block/Adminhtml/System/Config/Fieldset/Payment.php [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+     * @return string
+     * @SuppressWarnings(PHPMD.NPathComplexity)
+     */
+    protected function _getHeaderTitleHtml($element)
+    {
+        $html = '<div class="config-heading" >';
+
+        $groupConfig = $element->getGroup();
+
+        $disabledAttributeString = $this->_isPaymentEnabled($element) ? '' : ' disabled="disabled"';
```
Comment (payex-dev) [inline Block/Adminhtml/System/Config/Fieldset/Payment.php]: Codacy Issue found: [Avoid excessively long variable names like $disabledAttributeString. Keep variable name length under 20.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-core/pullRequest?prid=5236970)

**Comment by shahriar-vdc** [comment]
@asbjornu We need to tag it as 1.2.0 when this PR is merged to master

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Block/Adminhtml/System/Config/Fieldset/Payment.php  7
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-core/pullRequest?prid=5236970&bid=17165678)

File: Block/Adminhtml/System/Config/Fieldset/Payment.php (hunk 1 above)
Comment (payex-dev) [inline Block/Adminhtml/System/Config/Fieldset/Payment.php]: Codacy Issue found: [Avoid excessively long variable names like $disabledAttributeString. Keep variable name length under 20.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-core/pullRequest?prid=5236970)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- Block/Adminhtml/System/Config/Fieldset/Payment.php  7
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-core/pullRequest?prid=5236970&bid=17165678)

File: Block/Adminhtml/System/Config/Fieldset/Payment.php (hunk 1 above)
Comment (payex-dev) [inline Block/Adminhtml/System/Config/Fieldset/Payment.php:79]: Codacy Issue found: [Avoid excessively long variable names like $disabledAttributeString. Keep variable name length under 20.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-magento2-core/pullRequest?prid=5236970)

**Review by asbjornu — APPROVED** [review]

**Pull request merged**